lib/matplotlib/backends/backend_ps.py

```diff
@@ -1453,11 +1457,11 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
              'Here is the full report generated by LaTeX:\n%s '
              '\n\n' % (latexfile,
                        exc.output.decode("utf-8"))))
-    verbose.report(report, 'debug')
+    _log.debug(report)
 
     command = [str('dvips'), '-q', '-R0', '-o', os.path.basename(psfile),
                os.path.basename(dvifile)]
-    verbose.report(command, 'debug')
+    _log.debug(command)
```

Comment:
I'm actually surprised that this even works...

Reply:
Ummm....  It worked before?  I don't actually have a good way of checking if it works here.  Or can I just print an eps in debug mode?

Reply:
What I mean is that I didn't know you could pass a list to log.info and it'll just call str() on it itself (which seems like a misfeature but whatever).

It may make sense to set the log level to debug (while piping it to devnull :-)) on one of the travis job, to check that logging actually works :-) but that can wait too IMO.

Reply:
Yeah, these are all in `convert_psfrags`, and I don't even know how to get that fcn to call ;-)

Reply:
Why did you convert the one above to use `'%s'` and not this one?